app/controllers/admin/api/services_controller.rb

```diff
@@ -46,7 +46,9 @@ def index
   ##~ op.parameters.add :name => " ", :dataType => "custom", :paramType => "query", :allowMultiple => true, :description => "Extra parameters"
   #
   def create
-    service = current_account.create_service(service_params)
+    service = current_account.accessible_services.new(service_params)
```

Comment:
```suggestion
    service = current_account.services.new(service_params)
```

It is going to be accessible anyway so let's don't complicate our lifes 😄